drivers/include/periph/doc.txt

```diff
@@ -16,6 +16,64 @@
  * defines a standardized interface to access MCU peripherals that is not tied
  * to any specific vendor, platform or architecture.
  *
+ * # Initialization and Thread Safety
```

Comment:
Thread gets turned into a hyperlink thanks to automagic and mqtt paho.

Not sure whether there is anything we can do here -- escaping the word is probably a whack-a-mole...